src/plugins/ovp/services/JwplayerMediaSubscriber.js

```diff
@@ -23,7 +23,9 @@ export default class JwplayerMediaSubscriber extends EventSubscriber {
     data.jwplayerMediaId = node.get('jwplayer_media_id');
     data.jwplayerSyncEnabled = node.get('jwplayer_sync_enabled');
     data.jwplayerSyncedAt = node.has('jwplayer_synced_at')
-      ? moment(node.get('jwplayer_synced_at')).format('MMM DD, YYYY hh:mm A')
+      ? Microtime.fromString(String(`${node.get('jwplayer_synced_at')}000000`))
```

Comment:
@efjacobson - please adjust the util to format a Timestamp Type and use it: https://github.com/triniti/cms-js/blob/master/src/utils/convertReadableTime.js#L3

Reply:
@efjacobson - looks like it does handle the timestamps, so just use it: https://github.com/triniti/cms-js/blob/master/src/utils/convertReadableTime.js#L14